### DISCUSSION TITLE
Option to hide qty for some ingredients

### DISCUSSION THREAD

**Discussion by terrapinlogo** [mealie-recipes/mealie#1223] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
For some ingredients in a recipe a qty doesn't make much sense. It would be nice to have the option to hide the qty of the ingredient. 

eg. salt and pepper, condiments, ingredients that are "to taste"

**(Optional): Suggest A Solution**
User could optionally enter a qty of zero or leave the qty field blank and the qty for that ingredient would be hidden. Alternatively a check box to hide the qty.

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
This is already implemented, though there is a/some bugs

- https://github.com/hay-kot/mealie/issues/1237
